setup.py

```diff
@@ -44,11 +48,14 @@ def get_setup_version(reponame):
     import json, importlib
```

Comment:
This ``importlib`` import no longer appears to be used and is causing flake errors.

Reply:
Thanks. Must be a difference in flake8 versions as this wasn't caught in holoviews? I'll fix it in both places now.

Reply:
(it's a difference in how flake8 is run)

Reply:
Right, in holoviews it's only run on the holoviews and tests packages.

Reply:
When I inflict pyct on you, it will be the same everywhere, and everyone will hate me ;)